test/comments.ts

```diff
@@ -833,15 +833,15 @@ function runTestsForParser(parserId: any) {
       ["// comment", ";(function() {})();"].join(eol),
     );
   });
-  
+
```

Comment:
Would be better to add a new test in `patcher.ts`

Reply:
All 731 tests passed.

There is one "pending", which is `    - should parse and print fragments` in ` Esprima JSX Compatibility`, which has nothing to do with this PR.